spock-container-test-app/src/integration-test/resources/META-INF/services/grails.plugin.geb.ContainerGebTestDescription

```
@@ -0,0 +1 @@
+grails.plugin.geb.DefaultContainerGebTestDescription
```

I'm not sure we should keep this file. The default is loaded anyway, so keeping it might actually be misleading that it is mandatory.